### PR TITLE
UI: Update endpoint after show command.

### LINKED ISSUE #483: [PE-D] List item highlights not changing when show command is used
In the above screenshot, I clicked on the first list item to view it then change the item viewed using `show 10`, however, the highlighted item does not update. This could potentially be confusing to users.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: samuelfangjw/ped#7

### PR DESCRIPTION
**Pull request by ong6**
Fixes #483 
Closes #589 

Now add, edit, show and send will point to the Endpoint that they are using

Remove and run will unfocus the endpoint list to reduce confusion.

Sample:

sample

### CODE REVIEW

File: src/main/java/seedu/us/among/ui/MainWindow.java [hunk 1]
```diff
@@ -206,25 +206,58 @@ public EndpointListPanel getEndpointListPanel() {
         return endpointListPanel;
     }
 
+    /**
+     * Outputs the endpoint response for send and run commands to the
+     * result display and handles endpoint focus.
+     *
+     * @param commandResult the command entered
+     */
+    public void handelApiResponse(CommandResult commandResult) {
```
Comment (tjtanjin) [inline src/main/java/seedu/us/among/ui/MainWindow.java:215]: Typo in handle xD

File: src/main/java/seedu/us/among/ui/MainWindow.java [hunk 2]
```diff
@@ -206,25 +206,58 @@ public EndpointListPanel getEndpointListPanel() {
...
+        endpointListPanel.focusSelectedEndpoint(e);
+    }
+
+    /**
+     * Outputs the endpoint response for add, edit, remove, and show commands
+     * to the result display and handles endpoint focus.
+     *
+     * @param commandResult the command entered
+     */
+    public void handelEndpointResponse(CommandResult commandResult) {
```
Comment (tjtanjin) [inline src/main/java/seedu/us/among/ui/MainWindow.java:227]: Same as above

**Review by tjtanjin — APPROVED** [review]
LGTM, just typos to fix :P

File: src/main/java/seedu/us/among/ui/MainWindow.java (hunk 1 above)
Comment (ong6) [reply src/main/java/seedu/us/among/ui/MainWindow.java:215]: i wish i could speel

**Pull request merged**